README.md

```diff
@@ -714,4 +737,4 @@ Once you're ready to make a pull request: clone the repo, and install pnpm if yo
 
 If you're adding a feature, you should write a self-contained usage example in the form of a test, in [test/usage.test.ts](./test/usage.test.ts). This file is used to populate the bulk of this readme using [eslint-plugin-codegen](https://npmjs.com/package/eslint-plugin-codegen), and to generate an ["errors" test file](./test/errors.test.ts), which captures the error messages that are emitted for failing assertions by the typescript compiler. So, the test name should be written as a human-readable sentence explaining the usage example. Have a look at the existing tests for an idea of the style.
 
-After adding the tests, run `npm run lint -- --fix` to update the readme, and `npm test -- --updateSnapshot` to update the errors test. The generated documentation and tests should be pushed to the same branch as the source code, and submitted as a pull request. CI will test that the docs and tests are up to date if you forget to run these commands.
+After adding the tests, run `npm run lint -- --fix` to update the readme, and `npm test -- --update` to update the errors test. The generated documentation and tests should be pushed to the same branch as the source code, and submitted as a pull request. CI will test that the docs and tests are up to date if you forget to run these commands.
```

Comment:
Might be this is outdated. `--updateSnapshot` did not work for me.

Reply:
Try `--update` or just `-u`

Reply:
That’s right. `--update` does work as expected, hence I change this text.

Reply:
Yeah I think when we moved to Vitest recently we forgot to update this section of the docs.